### PR TITLE
Config to release the plugin on https://plugins.gradle.org/

### PR DESCRIPTION
**Pull request by ben-j69**
From https://github.com/klaxit/hidden-secrets-gradle-plugin/issues/14

Configuration and information about the plugin added to be able to release the project in plugins.gradle.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,26 +22,31 @@ This gradle plugin can be used with any Android project in Java or Kotlin.
...
 
-Get the latest version of the plugin from [releases](https://github.com/klaxit/hidden-secrets-gradle-plugin/releases).
-Copy `HiddenSecretsPlugin-X.Y.Z.jar` to your Android project in `/app/libs/` folder.
+## Using the plugins DSL
+In your app level `build.gradle`:
+
+```gradle
+plugins {
+    id "com.klaxit.hiddensecrets" version "X.Y.Z"
+}
```
Comment (Mazorius) [inline README.md:31]: I would recommend this:

```suggestion

// build.gradle
plugins {
    id "com.klaxit.hiddensecrets" version "X.Y.Z"
}

// build.gradle.kts
plugins {
    id("com.klaxit.hiddensecrets") version "X.Y.Z"
}
```

**Review by Mazorius — CHANGES_REQUESTED** [review]
I think this is the only think which would be nice to adapt.

File: README.md (hunk 1 above)
Comment (ben-j69) [reply README.md:31]: @Mazorius Thanks for your recommandation but we would like to keep the readme file as clear as possible. I think people using kts gradle files will know that they need to add parenthesis ?

Comment (Mazorius) [reply README.md:31]: Maybe we do not need to add anything of this 🤔 

As soon as you have published the plugin to plugins.gradle.org you get this: https://plugins.gradle.org/plugin/com.palantir.metric-schema

Maybe a link inside your readme is then enough information?

Comment (ben-j69) [reply README.md:31]: Yes I added a link to https://plugins.gradle.org/plugin/com.klaxit.hiddensecrets for all details.

**Pull request merged**